obal/data/playbooks/library/rpmspec_changelog_check.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+from ansible.module_utils.basic import AnsibleModule
+import os
+import subprocess
+import fnmatch
+
+def run_module():
+    module_args = dict(
+        directory=dict(type='str', required=True),
+    )
+
+    result = dict(
+        changed=False,
+        changelog_version_release='',
+        specfile_version_release=''
+    )
+
+    module = AnsibleModule(
+        argument_spec=module_args,
+        supports_check_mode=True
+    )
+
+    if module.check_mode:
+        return result
+
+    specdir = module.params['directory']
+
+    specfile = [ os.path.join(specdir, filename) for filename in os.listdir(specdir) if fnmatch.fnmatch(filename, "*.spec") ][0]
```

Comment:
Would it be easier to use `glob.glob(os.path.join(specdir, '*.spec'))`? I think that's equivalent.